streams/shared/src/main/scala/zio/stream/ZStream.scala

```diff
@@ -664,6 +664,12 @@ class ZStream[-R, +E, +A](val process: ZManaged[R, E, Pull[R, E, A]]) extends Se
     }
   }
 
+  /*** Chunks the stream with specifed chunkSize
+   * @parm chunkSize size of the chunk
+   */
+  def chunkN(chunkSize: Long): ZStream[R, E, List[A]] =
+    transduce(ZSink.collectAllN[A](chunkSize))
```

Comment:
How about:
```scala
def grouped(size: Long): ZStream[R,  E, List[A]]
def chunkN(size: Long): ZStream[R, E, Chunk[A]]
```

?

Reply:
`ZStream#grouped` being the same as `List#grouped`

Reply:
Yeah, I did'nt know that we have a Chunk (homework for me).

Reply:
@iravid I have question. How do I should implement this new `ChunkN` version? Is there `ZSink` operator for the Chunk result  ? Maybe, I should imlement one for this case (based on how ZSkink does that for the List)?

Reply:
Something like `ZSink.collectAllN[A](chunkSize).map(Chunk.fromIterable)`

Reply:
There's a much more efficient implementation that uses a pre-allocated buffer but we can do that in a follow-up if someone cares about the performance

Reply:
Ok. Tomorrow I will push changes for a review